### DISCUSSION TITLE
Create a metaverse

### DISCUSSION THREAD

**Discussion by micheltank** [SFTtech/openage#1466] [Ideas]
AOE is one of the most popular games in the past and could be a great candidate for a metaverse.

**Comment by roke-julian-lockhart** [comment]
> https://github.com/SFTtech/openage/discussions/1466#discussion-3934875

@micheltank, what the fuck does that mean?

**Comment by TobiasFP** [comment]
I would also, gracefully, like to enquire to the commonalities between this and that.

**Comment by dzremre** [comment]


**Reply by roke-julian-lockhart** [reply]
> https://github.com/SFTtech/openage/discussions/1466#discussioncomment-7989031

I have no idea what you're referring to, @dzremre.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/SFTtech/openage/discussions/1466#discussioncomment-7989302

Yeah, @dzremre, but what does that mean? You've not actually described anything. You've just listed things which AOE already appears to provide in the most vague terms possible.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/SFTtech/openage/discussions/1466#discussioncomment-7989333

@dzremre, I have no desire to delete it. I just want to know what you were excited for, because I have no idea of what you're referring to in https://github.com/SFTtech/openage/discussions/1466#discussioncomment-7989031. ...Surely you know what you meant?

**Comment by dzremre** [comment]


**Reply by roke-julian-lockhart** [reply]
> https://github.com/SFTtech/openage/discussions/1466#discussioncomment-7989367

@dzremre, you forgot to reply to https://github.com/SFTtech/openage/discussions/1466#discussioncomment-7989347 directly. Not a problem though. That doesn't explain what you mean, however. I'm just asking what you literally envision. Nobody knows what on Earth @micheltank meant when he posted https://github.com/SFTtech/openage/discussions/1466#discussion-3934875, so I'd like to know what you interpreted it as which excited you.
